### DISCUSSION TITLE
how to get parent link?

### DISCUSSION THREAD

**Discussion by skylai82** [schreibfaul1/ESP32-audioI2S#922] [General]
https://d2idp3hzkhjpih.cloudfront.net/out/v1/4b85d9c2bf97413eb0c9fd875599b837/index_3.m3u8

#EXTM3U
#EXT-X-VERSION:3
#EXT-X-TARGETDURATION:7
#EXT-X-MEDIA-SEQUENCE:11798101
#EXT-X-DISCONTINUITY-SEQUENCE:3073
#EXTINF:6.000,
index_3_11798101.ts?m=1661756343
#EXTINF:6.000,
index_3_11798102.ts?m=1661756343
#EXTINF:6.000,
index_3_11798103.ts?m=1661756343
#EXTINF:6.000,
index_3_11798104.ts?m=1661756343
#EXTINF:6.000,
index_3_11798105.ts?m=1661756343
#EXTINF:6.000,
index_3_11798106.ts?m=1661756343
#EXTINF:6.000,
index_3_11798107.ts?m=1661756343
#EXTINF:6.000,
index_3_11798108.ts?m=1661756343
#EXTINF:6.000,
index_3_11798109.ts?m=1661756343
#EXTINF:6.000,
index_3_11798110.ts?m=1661756343

**Comment by schreibfaul1** [comment]
this is a Video Stream

**Reply by skylai82** [reply]
https://radiomalaysia.online/#/live?catid=0&trackid=11
![image](https://github.com/user-attachments/assets/5fe25370-7511-467e-b403-b309bc04a585)
How did they do it?

**Reply by schreibfaul1** [reply]
![image](https://github.com/user-attachments/assets/82034cfb-2444-4e94-be46-0b281ed3f10b)

**Comment by skylai82** [comment]
https://www.rthk.hk/timetable/radio2
https://rthkradio2-live.akamaized.net/hls/live/2040078/radio2/index_64_a.m3u8
how to get the above parent link?
